src/components/ProjectMetaDataTable/SaveButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { useProjectProvider } from '@/context/ProjectProvider'
+import usePaymasterAttest from '@/hooks/project/usePaymasterAttest'
+import { toast } from '../ui/use-toast'
+
+const SaveButton = () => {
+  const { attest } = usePaymasterAttest()
+  const { setCreatingStatus, animationUrl, image } = useProjectProvider()
+
+  const handleClick = async () => {
+    if (!animationUrl || !image) return
```

Comment:
can I save if I select EITHER `image` OR `audio`???
- expected `yes I can`
```suggestion
    if (!(animationUrl || image)) return
```